r/R/utilities_development.R

```diff
@@ -0,0 +1,11 @@
+#' Loads Experiment data
+#'
+#' @param exp_id
+#'
+#' @return
+#' @export
+#'
+#' @examples
+load_experiment_data <- function(exp_id = "test") {
```

Comment:
that looks like a function for setting up testing, I would move it to be under `tests/`